### ISSUE TITLE
Proposing Triage Team Meetings ( additional, supplementary to TC )

### ISSUE THREAD

**Issue by UlisesGascon** [expressjs/discussions#276]
As agreed on the last TC meeting, let's find a good slot to meet once a month to discuss traige and moderation topics. I created this doodle to find a good spot:

https://doodle.com/meeting/participate/id/bY0NEBWa

The Doodle is focused on next week (but the idea to use it as a generic availability reference); I will compare the most popular times against the Node.js and Foundation calendars to avoid double-booking, so don't worry if you choose an option that overlaps with a biweekly calendar or similar.

Note: as we are in very different timezones maybe we can have the meetings in 2 different slots to cover more regions (like early and late europe)

cc: @expressjs/triagers

**Comment by IamLizu** [comment]
Awesome 🎉 

I have voted for three different days. I hope we can fix a time or two that works best for all of us 👍

**Comment by UlisesGascon** [comment]
![Screenshot from 2024-10-08 14-35-11](https://github.com/user-attachments/assets/8bde0d09-a78a-473f-8a1e-e8131ee9ed13)

Seems like the best options are (in UTC +2):
- Monday 21:00 to 22:00 (just before the TC meeting)
- Thursday 16:00 to 17:00 (just to avoid conflicting with Node.js Security WG Meeting.)


So I suggest that we can make a meeting once a month and alternate between both options.

So, Next meeting on Monday, Oct 28th just before the TC Meeting. @bensternthal can you include it in the OpenJS Calendar as "Express triage meeting?"

**Comment by bensternthal** [comment]
@UlisesGascon happy to help, a few questions

- private or public?
- auto record
- who should be included on the invite

**Comment by UlisesGascon** [comment]
It should be public with option to record but not auto-record.

**Comment by bensternthal** [comment]
@UlisesGascon Would it be possible to consider not alternating the meetings? In the past, we’ve encountered challenges with this approach when a meeting needs to be canceled or rescheduled. When that happens, it often leads to confusion for attendees since both events have to be rescheduled. Having a consistent schedule would help avoid this issue. Do let me know.

**Comment by UlisesGascon** [comment]
well.. let's schedule for Monday, Oct 28th and in that meeting we can decide what to do with the rescheduling :+1:

**Comment by UlisesGascon** [comment]
@bensternthal can you confirm if it was added into the calendar? I can't find it :thinking:

**Comment by bensternthal** [comment]
Yep you should have it in your calendar. I just copied to the OpenJS public calendar.

**Comment by wesleytodd** [comment]
We discussed this on the meeting today. 

I now have access to the project in the LFX dashboard to create meetings, and @UlisesGascon is going to reach out to @bensternthal about getting setup with the same access. We agreed this should be given to all the TC members as we do with other administrative access. In the next few weeks I am going to do an audit of this and make a list of all the things so we can sort it out on a more permanent basis.

Additionally we said we would ask @bensternthal if we could manage the ones on the OpenJS calendar by moving them into the org. I thought about after the meeting that maybe that would not work if they are tied to the project in LFX, but maybe @bensternthal can comment on that as well?

In the mean time, I think this meeting is on the books somewhere and so I can go ahead and close this issue.

**Comment by bensternthal** [comment]
Ulises has just been added and I requested that the following meetings be moved to the Express project so that you can manage them.

Express TC Meeting
Express Triage Team Meeting
Express Security WG

**Comment by wesleytodd** [comment]
That won't remove them from the OpenJS calendar right? Just making sure we know because I rely on that calendar lol.

**Comment by bensternthal** [comment]
It _should not,_ however we might want to move to the LFX generated public calendar at some point. Maintaining the OpenJS google calendar has always been manual.
